docs/workflows/publish.rst

```diff
@@ -41,7 +41,7 @@ Response::
       "pulp_href": "/pulp/api/v3/distributions/python/pypi/4839c056-6f2b-46b9-ac5f-88eb8a7739a5/",
       "pulp_created": "2021-03-09T04:36:48.289737Z",
       "base_path": "foo",
-      "base_url": "https://pulp3-source-fedora33.localhost.example.com/pulp/content/foo/",
+      "base_url": "/pypi/foo/",
```

Comment:
Does `base_url` no longer return the full url?

Reply:
Yeah, I don't know how to find the hostname to construct the full url.